### PR TITLE
IDGEN-68 Log Entry Search Handler

### PR DESCRIPTION
**Pull request by cnagadya**
## JIRA TICKET NAME:

[IDGEN-68 Create Log Entries Search](https://issues.openmrs.org/browse/IDGEN-77)

## SUMMARY:
Search Handler for the ID Generation Logs

### CODE REVIEW

File: omod/src/main/java/org/openmrs/module/rest/search/LogEntrySearchHandler.java [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+        List<LogEntry> logEntries = new ArrayList<LogEntry>();
+
+        IdentifierSource logSource = iss.getIdentifierSourceByUuid(source);
+        Date dateFrom = fromDate != null ? (Date) ConversionUtil.convert(fromDate, Date.class) : null;
+        Date dateTo = toDate != null ? (Date) ConversionUtil.convert(toDate, Date.class) : null;
+        UserService service = Context.getUserService();
+        User user = generatedBy != null ? service.getUserByUuid(generatedBy) : null;
+
+        logEntries = iss.getLogEntries(logSource, dateFrom, dateTo, identifier, user, comment);
```
Comment (dkayiwa) [inline omod/src/main/java/org/openmrs/module/rest/search/LogEntrySearchHandler.java]: identifierSourceService is clearer than iss

**Comment by cnagadya** [comment]
@dkayiwa, the requested changes have been made

**Comment by dkayiwa** [comment]
@cnagadya have you added all the classes required by the ticket description?

**Comment by cnagadya** [comment]
Please expound on your questin @dkayiwa, do you mean classes for the search handler? Because its the only one I have competed so far

**Comment by dkayiwa** [comment]
@cnagadya ok just go on with the rest of the classes.

**Comment by cnagadya** [comment]
Alright

**Pull request closed without merging**